tests/framework/microvm.py

```diff
@@ -926,6 +926,21 @@ def ssh(self):
         """Return a cached SSH connection on the 1st interface"""
         return self.ssh_iface(0)
 
+    def verify_if_guest_is_up(self, up=True):
```

Comment:
How about `raise_if_down(self, timeout=1)` for name?

The timeout is now infinity, that's why some tests time out after 300s (our test limit). By having it centralized here we could lower it so test could fail faster.

Reply:
That's true! The slowest guest boot time metric is about 600,000 us (0.6s) on m7g.metal / 4.14 host with a single agent, so the default timeout might need to be greater than 1s though.